src/block-management/block-toolbar.js

```diff
@@ -59,8 +59,9 @@ export class BlockToolbar extends Component<PropsType> {
 					<Toolbar>
 						<ToolbarButton
 							label={ __( 'Add block' ) }
-							icon="insert"
+							icon={ ( <Dashicon icon="plus-alt" style={ styles.addBlockButton } color={ styles.addBlockButton.color } /> ) }
 							onClick={ onInsertClick }
+							iconProps={ { style: styles.addBlockButton, color: styles.addBlockButton.color } }
```

Comment:
Do we still need this line?
I can't find any usage of `iconProps` in `ToolbarButton`

Reply:
Right! It's not needed anymore.
Thanks for noticing 🙏